test/integration/integration-test.ts

```diff
@@ -513,7 +505,7 @@ describe('integration tests', function () {
   it('generateWallet', function () {
     const newWallet = this.api.generateAddress()
     assert(newWallet && newWallet.address && newWallet.secret)
-    assert(isValidAddress(newWallet.address))
+    assert(isValidClassicAddress(newWallet.address))
```

Comment:
Ah that's what the problem was - I was getting errors about this and wasn't sure why.